freebsd/kern/kern_event.c

```diff
@@ -2221,7 +2221,11 @@ kqueue_wakeup(struct kqueue *kq)
 			kq->kq_state &= ~KQ_SEL;
 	}
 	if (!knlist_empty(&kq->kq_sel.si_note))
+#ifndef FSTACK
 		kqueue_schedtask(kq);
```

Comment:
gcc build error :`kern_event.c:1676:1: error: ‘kqueue_schedtask’ defined but not used`
May be you also use `#ifndef FSTACK` and '#endif' to protect the definition of `kqueue_schedtask` in line 1676.

Reply:
ok